extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```diff
@@ -157,77 +156,35 @@ describes.sandboxed('amp-ad-network-doubleclick-impl', {}, () => {
       });
     });
 
-    it('without signature', () => {
-      const headers = {
+    it('without analytics', () => {
```

Comment:
Nit: Could this test name be made more clear? I get that it's no *less* clear than 'without signature' was, but still not as descriptive as it could be. Ditto 'with analytics'.